src/utils.rs

```diff
@@ -76,3 +77,34 @@ pub fn find_workspace(manifest: &Path, name: &str) -> Result<Option<PathBuf>, Er
     }
     Ok(None)
 }
+
+/**
+ * Search for .cargo/config.toml file
+ *
+ * - Receive the workspace root path
+ */
+pub fn find_cargo_config(path: &Path) -> Result<Option<PathBuf>, Error> {
+    let path = dunce::canonicalize(path)?;
+    for config_path in path
+        .ancestors()
+        .map(|dir| dir.join(".cargo/config.toml"))
+        .filter(|dir| dir.is_file())
+    {
+        return Ok(Some(config_path));
+    }
+    Ok(None)
+}
+
+pub fn get_target_dir_name(path: &Path) -> Result<String, Error> {
+    if let Some(config_path) = find_cargo_config(&path).unwrap() {
+        let config = Config::parse_from_toml(&config_path)?;
+        if let Some(build) = config.build.as_ref() {
+            if let Some(target_dir) = &build.target_dir {
+                return Ok(target_dir.clone());
+            }
+        }
+    } else {
+        return Ok("target".to_string());
+    }
+    Err(Error::InvalidConfig)
```

Comment:
Not a fan of return, can be:

```suggestion
        }
        Err(Error::InvalidConfig)
    } else {
        Ok("target".to_string())
    }
```